weave-js/src/common/util/cookie.ts

```diff
@@ -157,3 +157,14 @@ function getFirebaseCookiesObject(): Struct<string> {
     return {};
   }
 }
+
+export const generateCookieString = () => {
```

Comment:
why not put this in the core pr where it is used?

Reply:
it looks like all the cookie helpers were in this file